govcd/orgvdcnetwork.go

```diff
@@ -184,7 +186,7 @@ func (vdc *Vdc) CreateOrgVDCNetWorkBasic(networkConfig *types.OrgVDCNetwork) (Ta
 			for _, taskItem := range orgVDCNetwork.OrgVDCNetwork.Tasks.Task {
 				return Task{taskItem, vdc.client}, nil
 			}
-			return Task{}, fmt.Errorf("[CreateOrgVDCNetWorkBasic] No suitable task found")
+			return Task{}, fmt.Errorf("[CreateOrgVDCNetWork] No suitable task found")
```

Comment:
Knowing function name here is useful, but if someone changes the name of the function (one more time), the function name in the error message may be left out :) Please see if using runtime reflection to get the function name could be a stable alternative (reflections are not always straightforward).

Reply:
This could be easy solved by: https://github.com/gogap/aop

Reply:
I have added a function that returns the current function name, so that this message will always return the current function. BTW, the current function name is misspelled: the "w" should be lowercase, and so this change comes at the right moment.